src/CountSpChar.java

```diff
@@ -0,0 +1,24 @@
+
+public class CountSpChar {
+	void displaySpCharCount(String str) {
+		String str1 = str;
+
+		str1 = str1.replace('[', ' ');
+		str1 = str1.replace('{', ' ');
```

Comment:
replace method can be called in a chain like below code.

str1.replace("[","").replace("{","") so on...

Reply:
Code changes based on given your requirement.